test/functional/feature_notifications.py

```diff
@@ -18,7 +18,7 @@
 # Windows disallow control characters (0-31) and /\?%:|"<>
 FILE_CHAR_START = 32 if os.name == 'nt' else 1
 FILE_CHAR_END = 128
-FILE_CHAR_BLOCKLIST = '/\\?%*:|"<>' if os.name == 'nt' else '/'
+FILE_CHAR_BLACKLIST = '/\\?%*:|"<>' if os.name == 'nt' else '/'
```

Comment:
please end this pointless cycle and change it to `FILE_CHARS_DISALLOWED``

Reply:
FILE_CHARS_BANLIST

Reply:
I changed it to FILE_CHARS_DISALLOWED.